cmd/nginx/flags.go

```diff
@@ -127,6 +127,10 @@ func parseFlags() (bool, *controller.Configuration, error) {
 
 		syncRateLimit = flags.Float32("sync-rate-limit", 0.3,
 			`Define the sync frequency upper limit`)
+
+		publishStatusAddress = flags.String("publish-status-address", "",
+			`Address to expose the ingress controllers. The controller will set the
+		endpoint records on the ingress using this address.`)
```

Comment:
Can you rephrase this? This could lead to understanding nginx will bind to the IP address.

Reply:
Thanks, I have updated it.